src/heap_utils.cc

```diff
@@ -231,19 +234,178 @@ class BufferOutputStream : public v8::OutputStream {
   std::unique_ptr<JSString> buffer_;
 };
 
+namespace {
+class FileOutputStream : public v8::OutputStream {
+ public:
+  explicit FileOutputStream(FILE* stream) : stream_(stream) {}
+
+  virtual int GetChunkSize() {
```

Comment:
Sorry for missing this on the first round but can you add `override` to the virtual methods?